src/app/(views)/managed-courses/[courseId]/modules/page.tsx

```diff
@@ -51,6 +75,83 @@ export default function Modules() {
           </tr>
         ))}
       </tbody>
-    </table>
+      </table> */}
+      <CContainer className="mt-4">
+        <CCol>
+          <div className="d-flex justify-content-end align-items-center mb-4">
+            <CButton
+              color="dark"
+              className="text-white"
+              as="a"
+              href={`/managed-courses/${courses[0]?.id}/modules/create`}
+            >
+              <CIcon icon={cilPlus} className="me-2" />
+              Create Module
+            </CButton>
+          </div>
+
+          <>
+            <CTable striped>
+              <CTableHead>
+                <CTableRow className="text-secondary">
+                  <CTableHeaderCell>
+                    <small>ID</small>
+                  </CTableHeaderCell>
+                  <CTableHeaderCell>
+                    <small>Title</small>
+                  </CTableHeaderCell>
+                  <CTableHeaderCell>
+                    <small>Type</small>
+                  </CTableHeaderCell>
+                  <CTableHeaderCell>
+                    <small>Actions</small>
+                  </CTableHeaderCell>
+                </CTableRow>
+              </CTableHead>
+              <CTableBody>
+                {courses[0]?.modules.map((module) => (
```

Comment:
If there are no modules, add a table row saying "No modules" or something along the lines thereof.